app/controllers/miq_ae_class_controller.rb

```diff
@@ -2554,14 +2554,12 @@ def domain_overrides
   end
 
   def get_session_data
-    @layout     = "miq_ae_class"
```

Comment:
You have to create a class method `session_key_prefix` and move `"miq_ae_class"` there to be able to remove this method. `get_session_data` in the mixin is calling it (see https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/mixins/generic_session_mixin.rb#L7 ).